pom.xml

```diff
@@ -114,6 +114,12 @@
       <scope>test</scope>
       <version>1.9.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.sonar</groupId>
```

Comment:
Why is this dependency needed? "org.codehaus.sonar" is the old name and in line 69 we have a dependency to the same artifact, just a bit higher version and "org.sonarsource.sonarqube" (the new name). I think the old stuff should no longer be needed (to my understanding - maybe additional code adjustments are needed)